include/openssl/core.h

```diff
@@ -42,8 +42,7 @@ struct ossl_dispatch_st {
     void (*function)(void);
 };
 
-# define OSSL_DISPATCH_END \
-    { 0, NULL }
+# define OSSL_DISPATCH_END { 0, NULL }
```

Comment:
Do we need this change?

Reply:
Removed - I was trying to fix code style nits.